Code/Framework/AzToolsFramework/AzToolsFramework/UI/Outliner/EntityOutlinerListModel.cpp

```diff
@@ -764,10 +765,18 @@ namespace AzToolsFramework
         return canHandleData;
     }
 
-    bool EntityOutlinerListModel::CanDropMimeDataAssets(const QMimeData* data, Qt::DropAction /*action*/, int /*row*/, int /*column*/, const QModelIndex& /*parent*/) const
+    bool EntityOutlinerListModel::CanDropMimeDataAssets(const QMimeData* data, Qt::DropAction /*action*/, int /*row*/, int /*column*/, const QModelIndex& parent) const
     {
         using namespace AzToolsFramework;
-        
+
+        // Disable dropping assets on closed container entities.
+        AZ::EntityId parentId = GetEntityFromIndex(parent);
+        if (auto containerEntityInterface = AZ::Interface<ContainerEntityInterface>::Get();
```

Comment:
suggestion: As `auto containerEntityInterface = AZ::Interface<ContainerEntityInterface>::Get();` is called twice we could pull this out and use in both conditionals here and below?

Reply:
The calls are in separate functions, so I'd have to move it to a member. I was considering doing it, but the calls are very sporadic so I didn't want to complicate the class yet. If I need to add more calls for other bugfixes, I'll consider moving the Get so it's only done once :)

Reply:
Sounds good 👍